src/dune_lang/package_variable_name.ml

```diff
@@ -49,13 +42,65 @@ let version = of_string "version"
 let name = of_string "name"
 let build = of_string "build"
 let post = of_string "post"
-let one_of t xs = List.mem xs ~equal t
 let dev = of_string "dev"
+let one_of t xs = List.mem xs ~equal t
```

Comment:
This is moved for cleanup reasons?

Reply:
Yeh it's just so that all of the variable names are next to each other.